README.md

```diff
@@ -1,2 +1,17 @@
 # HW01
-Edit the readme
+
+### L. E. Hunter - Biography
+
+Hello, everyone! My name is Laura. I'm an incoming **Integrative Biology** student interested in human evolution. I graduated from Columbia University in 2017 and have been working at a [neuroscience lab](https://www.gottlieblab.com/) at the [Zuckerman Institute for Mind, Brain, and Behavior](https://zuckermaninstitute.columbia.edu/) ever since. In my spare time, I work as an emergency medical technician and volunteer as a fossil and primatology explainer at the American Museum of Natural History (although the museum stuff has been on hold since March due to COVID). I also like to paint. Check out my painting of the skeleton of the animal I studied while working at the Zuckerman Institute:
```

Comment:
I think this review is a little harder to actually review since there is not a whole lot of code so instead I just want to comment on your bio because I think it is so cool what you do. Fossil volunteer sounds so fun and awesome!